packages/@aws-cdk/aws-lambda/lib/lambda.ts

```diff
@@ -171,6 +172,15 @@ export interface FunctionProps {
    * @default undefined X-Ray tracing disabled
    */
   tracing?: Tracing;
+
+  /**
+   * A list of layers to add to the function's execution environment. You can configure your Lambda function to pull in
+   * additional code during initialization in the form of layers. Layers are packages of libraries or other dependencies
+   * that can be used by mulitple functions.
+   *
+   * @default no layers
+   */
+  layerVersions?: LayerVersionRef[];
```

Comment:
I think “layers” is expressive enough. The type indicates that those are layer versions